twake/frontend/src/app/scenes/Client/app.less

```diff
@@ -1,4 +1,4 @@
-@import '~components/constants.scss';
+@import '~antd/dist/antd.less';
 
```

Comment:
I don't want to use less except for main Antd theme. Can we keep scss file extension ?

Reply:
Done